src/RequestInspector.tsx

```diff
@@ -64,6 +64,7 @@ function ProofDisplay({ proof } : { proof: Proof }) {
     const capabilities = proof.capabilities.map(capability => <CapabilityDisplay capability={capability} />)
     const proofs = proof.proofs.map((proof) => <ProofDisplay proof={proof}/>)
     const index: Record<string, ReactNode> = {
+      'Root CID': proof.cid.toString(),
```

Comment:
suggestion: please put this in `<ShortenAndScroll>` like the values below to avoid blowing this out sideways too much